src/appComponents/sessions/SessionsController.js

```diff
@@ -51,30 +52,30 @@ sessionsModule.controller("SessionsController",
                                     + openevent.totalDays);
                                 continue;
                             }
-                            sc.Sessions[dayDiff].push(response.data.sessions[j]);
-                            $sessionStorage.days[dayDiff].sessions = sc.Sessions[dayDiff];
+                            sec.Sessions[dayDiff].push(response.data.sessions[j]);
+                            $sessionStorage.days[dayDiff].sessions = sec.Sessions[dayDiff];
                             $sessionStorage.days[dayDiff].sessions.sort(SortUtils.sortBy(
                             	"begin",
                             	false,
                             	function(a){return a;}
                             	));
                         }
-                        sc.showLoaders = false;
-                        sc.Days = $sessionStorage.days;
+                        sec.showLoaders = false;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 10 space characters but found 12.](https://www.codacy.com/app/triplez/open-event-webapp/file/2445549891/issues/source?bid=3233857&fileBranchId=3245777#l63)